### ISSUE TITLE
If Electrs is installed, wait for it to be up and running before starting BTC-RPC-Explorer

### ISSUE THREAD

**Issue by got3nks** [openoms/raspiblitz-extras#4]
If electrs is installed, BTC-RPC-Explorer should wait for it to be online before starting, or it won't work. I could reproduce this issue every time I rebooted my node and I needed to restart BTC-RPC-Explorer after Electrs was fully up and running.

```
Dec 08 11:26:37 btc-rpc-explorer[2263]: 2019-12-08T10:26:37.675Z btcexp:error Error 21307ws70sg: TypeError: Cannot read property 'result' of undefined, json: {}
Dec 08 11:26:37 btc-rpc-explorer[2263]: 2019-12-08T10:26:37.675Z btcexp:error Stack: TypeError: Cannot read property 'result' of undefined
Dec 08 11:26:37 btc-rpc-explorer[2263]:     at /usr/local/lib/nodejs/node-v10.16.0-linux-armv7l/lib/node_modules/btc-rpc-explorer/app/api/electrumAddressApi.js:119:26
```

As a solution, I changed the systemd service unit:

`ExecStart=/home/bitcoin/btc-rpc-explorer.run.sh`

btc-rpc-explorer.run.sh code:

```
#!/bin/bash

echo "Waiting Electrs on port 50001..."

while ! nc -z localhost 50001; do
  sleep 0.1
done

echo "Electrs started, launching BTC-RPC-Explorer..."

/usr/local/lib/nodejs/node-v10.16.0-linux-armv7l/bin/btc-rpc-explorer
```

**Comment by openoms** [comment]
excellent suggestion thanks, will update!

**Comment by openoms** [comment]
The new changes fixing this issue are waiting to be merged to the RaspiBlitz 1.4. The instructions in repo are modified to help testing.
